### PR TITLE
More refinery weapon description updates

### PR DESCRIPTION
**Pull request by tired-wired**
## About The Pull Request
Elaborates on low to mid tier refinery weapon descriptions.

## Why It's Good For The Game
It's just some text.

## Changelog
:cl:
tweak: tweaked fixer weapon descriptions and fixed a tense mistake
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/non_abnormality/miscfixer.dm [hunk 1]
```diff
@@ -1,8 +1,7 @@
 //Wedge - Grade 4 with two range.
 /obj/item/ego_weapon/city/wedge
 	name = "wedge office spear"
-	desc = "A black, ornate spear used by the wedge office."
-	icon_state = "wedge"
```
Comment (Kitsunemitsu) [inline code/game/objects/items/ego_weapons/non_abnormality/miscfixer.dm:5]: Accidentally deleted the sprite.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Accidentally deleted the sprite for wedge office.

File: code/game/objects/items/ego_weapons/non_abnormality/miscfixer.dm (hunk 1 above)
Comment (tired-wired) [reply code/game/objects/items/ego_weapons/non_abnormality/miscfixer.dm:5]: oh geez, whoops. i'll put that back in right away

**Review by Kitsunemitsu — APPROVED** [review]
Approved.

**Pull request merged**